modules/tensor_mechanics/src/actions/DomainIntegralAction.C

```diff
@@ -304,8 +302,11 @@ DomainIntegralAction::act()
     if (_use_crack_front_points_provider)
     {
       params.set<UserObjectName>("crack_front_points_provider") = _crack_front_points_provider;
-      params.set<unsigned int>("number_points_from_provider") = _number_crack_front_points;
+      params.set<unsigned int>("number_points_from_provider") =
+          getParam<unsigned int>("number_points_from_provider");
```

Comment:
You might use the following method to clean up this block of code:
https://github.com/idaholab/moose/blob/devel/framework/include/utils/InputParameters.h#L576-L577

Reply:
Nice tool. Thanks!